device/sd_flasher.go

```diff
@@ -33,118 +33,126 @@ type sdFlasher struct {
 
 // MountImg is a method to attach image to loop and mount it
 func (d *sdFlasher) MountImg(loopMount string) error {
-	logrus.Debug("Attaching an image")
-	command := fmt.Sprintf("losetup -f -P %s", filepath.Join(constants.TMP_DIR, d.img))
-	out, eut, err := d.conf.SSH.Run(command)
-	if err != nil {
-		logrus.Error("[-] Error when execute: ", command, eut)
+	log.Debug("Attaching an image")
+	command := fmt.Sprintf("losetup -f -P %s", help.AddPathSuffix("unix", constants.TMP_DIR, d.img))
+	if out, eut, err := d.conf.SSH.Run(command); err != nil {
+		log.Error("[-] Error when execute: ", command, eut)
 		return err
+	} else if strings.TrimSpace(out) != "" || strings.TrimSpace(eut) != "" {
+		log.Debug(out, eut)
 	}
-	logrus.Debug(out, eut)
 
-	logrus.Debug("Creating tmp folder")
+	log.Debug("Creating tmp folder")
 	command = fmt.Sprintf("mkdir -p %s", constants.MountDir)
-	out, eut, err = d.conf.SSH.Run(command)
-	if err != nil {
-		logrus.Error("[-] Error when execute: ", command, eut)
+	if out, eut, err := d.conf.SSH.Run(command); err != nil {
+		log.Error("[-] Error when execute: ", command, eut)
 		return err
+	} else if strings.TrimSpace(out) != "" || strings.TrimSpace(eut) != "" {
+		log.Debug(out, eut)
 	}
-	logrus.Debug(out, eut)
 
 	if loopMount == "" {
 		command = fmt.Sprintf("ls /dev/loop0p*")
-		out, eut, err = d.conf.SSH.Run(command)
-		if err != nil {
-			logrus.Error("[-] Error when execute: ", command, eut)
-			return err
-		}
-		logrus.Debug(out, eut)
-
 		compiler, _ := regexp.Compile(`loop0p[\d]+`)
-		opts := compiler.FindAllString(out, -1)
-
-		if len(opts) == 0 {
-			return errors.New("Cannot find a mounting point")
+		if out, eut, err := d.conf.SSH.Run(command); err != nil {
+			log.Error("[-] Error when execute: ", command, eut)
+			return err
+		} else {
```

Comment:
__[golint]__
* [golint] if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)

<!-- Comment by [SideCI](https://sideci.com) -->